smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -44,6 +44,62 @@ contract MixinTransfer is
   // This is calculated as `keyPrice * transferFeeBasisPoints / BASIS_POINTS_DEN`.
   uint public transferFeeBasisPoints;
 
+  /**
+  * @notice Allows the key owner to share their key (parent key) by
+  * transferring a portion of the remaining time to a new key (child key).
+  * @param _from The owner of the parent key
+  * @param _to The recipient of the shared key
+  * @param _tokenId the key to share
+  * @param _timeShared The amount of time shared
+  */
+  function shareKey(
+    address _from,
+    address _to,
+    uint _tokenId,
+    uint _timeShared
+  ) public
+    onlyIfAlive
+    hasValidKey(_from)
+    onlyKeyOwnerOrApproved(_tokenId)
+  {
+    require(_to != address(0), 'INVALID_ADDRESS');
+
+    Key storage fromKey = keyByOwner[_from];
+    Key storage toKey = keyByOwner[_to];
+    uint IdTo = toKey.tokenId;
```

Comment:
minor: style convention, not sure we ever start a variable with a capital letter..
where's ESlint on this?!  heh I still wonder if that's actually running at all sometimes.

Reply:
oops, my bad. fixed.